packages/expo-router/src/useFocusEffect.ts

```diff
@@ -28,13 +28,13 @@ export type EffectCallback = () => undefined | void | (() => void);
  *     // Callback should be wrapped in `React.useCallback` to avoid running the effect too often.
  *     useCallback(() => {
  *       // Invoked whenever the route is focused.
- *       console.log('Hello, I'm focused!');
+ *       console.log("Hello, I'm focused!");
  *
  *       // Return function is invoked whenever the route gets out of focus.
  *       return () => {
  *         console.log('This route is now unfocused.');
  *       };
- *     }, []);
+ *     }, []),
```

Comment:
```suggestion
 *     }, []);
```

I'm afraid this wasn't supposed to be a comma.